Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    rotoscope (0.1.0)
+    rotoscope (0.1.1)
```

Comment:
I saw this change from just doing `bundle install` without any other changes.  Why do we even check in the Gemfile.lock for a library? Usually it is git ignored.